db/seeds.rb

```diff
@@ -12,6 +12,16 @@
   puts 'ユーザーの初期データインポートに成功しました。'
 end
 
+EMAIL = 'admin@example.com'
+PASSWORD = 'password'
+PASSWORD_CONFIRMATION = 'password'
```

Comment:
定数は同じ名前を使用するとエラーがでます。パスワードは同じなので，メールアドレスだけ決めておきましょうか。

```suggestion
ADMIN_EMAIL = 'admin@example.com'
```